rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -348,21 +350,24 @@ def run_all_post_decoding_rules(
         assert self._check_correct_types(decoded_events)
         return decoded_events
 
-    def decode_transaction(
+    def _decode_transaction(
```

Comment:
Why are you doing this? Why add the underscore here and make this a "private" function, and force such a big change in lines in an already big PR?

This function is called mostly in tests but over 80 times. What do we gain by you adding this underscore at this point, in this PR?

It's just adding another 80 lines for me to check.

Reply:
I guess you have already marked the files that are touched by this change as "Viewed", so in order to not reset that "Viewed" status I will keep it as it is right now.

Reply:
Not really. As I said I stopped reviewing the PR before due to the size of the irrelevant diff. But whatever I am not gonna let this PR block the frontend and the release anymore.

Please, this is something to keep in mind for future PRs. Keep things short and simple. And do not add any unecessary changes like this in a PR.